engine/order_manager.go

```diff
@@ -388,6 +388,34 @@ func (m *OrderManager) GetOrdersSnapshot(s order.Status) ([]order.Detail, time.T
 	return os, latestUpdate
 }
 
+// GetOrdersFiltered returns a snapshot of all orders in the order store.
+// Filtering is applied based on the order.Filter unless entries are empty
+func (m *OrderManager) GetOrdersFiltered(f *order.Filter) ([]order.Detail, time.Time, error) {
```

Comment:
for now, it might be an idea to drop the time.Time return, it doesn't seem like it's being used.

Reply:
implemented